decksite/templates/cardstable.mustache

```diff
@@ -6,23 +6,33 @@
             <th class="n" data-sortInitialOrder="desc"># Decks</th>
             <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">Record</th>
             <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">Win %</th>
-            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">①</th>
-            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">⑧</th>
-            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">5–0s</th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="Tournament wins">①</abbr>
+            </th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="Tournament top-8s">⑧</abbr>
+            </th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
+                <abbr title="League 5-0 runs">5–0s</abbr>
+            </th>
+            <th class="n" data-breakpoints="xs" data-sortInitialOrder="desc">
```

Comment:
We used to display this information back when we displayed "this season" and "all time" alongside one another but dropped it on the thinking that in general you'd rather see something simpler and faster. I can see the argument that it is interesting enough to include, though. Should the values be "main deck" and "sideboard" rather than "# decks" and "# SBs". Also "# SBs" is kind of an ugly construction but might be a necessary evil. I can't think of anything short and better but that again makes me question the premise.

Reply:
My original motivation for adding this was to help myself and others build their sideboards. The /cards/ page could theoretically help by showing what cards other people are playing in their sideboards. However, I wanted to keep it All Decks and Sideboards (rather than #main and #side) to preserve the chart's sorting as overall cards by popularity. 

Totally open to #SBs being changed to something else, part of the reason I added the <abbr> tags is because I didn't like how that looked.

Reply:
Yeah I think that's a very real motivation. I have experienced it myself. I'm trying to think if there's a better way to satisfy that need. I can envisage a /cards/sideboard/ but that's pretty weird. Maybe this is the best way. Worth discussing in #code or even #general to see if anyone has a smart idea? An additional column is not such a huge thing especially with gzip but it's worth canvassing a few ideas maybe.

Reply:
I wonder how bad "# sideboards" looks. It's very long compared to most of the headings. "# side"??